modules/openy_gc_auth/modules/openy_gc_auth_reclique/src/RecliqueClientService.php

```diff
@@ -29,12 +37,15 @@ class RecliqueClientService {
   /**
    * RecliqueClientService constructor.
    *
+   * @param \Drupal\Core\Logger\LoggerChannelFactory $loggerFactory
+   *   LoggerChannelFactory instance.
    * @param \Drupal\Core\Config\ConfigFactoryInterface $configFactory
    *   ConfigFactoryInterface instance.
    * @param \GuzzleHttp\Client $client
    *   Guzzle client.
    */
-  public function __construct(ConfigFactoryInterface $configFactory, Client $client) {
+  public function __construct(LoggerChannelFactory $loggerFactory, ConfigFactoryInterface $configFactory, Client $client) {
+    $this->loggerFactory = $loggerFactory->get('openy_gc_auth_reclique');
```

Comment:
it's not a factory then, when you are getting a logger via get method
it's a logger

Reply:
Makes sense - thanks!  I committed Anatoliy's suggested change, so the next build will include this.